README.md

```diff
@@ -1,78 +1,97 @@
 - [License](#license)
 
-## Security
+## Background
 
-TBD
+See related specs:
 
-## Background
+- [Verifiable Credentials Data Model](https://github.com/w3c/vc-data-model)
+- [VCDM v1.1](https://www.w3.org/TR/vc-data-model/)
+- [VCDM v2.0](https://www.w3.org/TR/vc-data-model-2.0/)
 
-See also (related specs):
+## Security
 
-* [Verifiable Credentials Data Model v1](https://github.com/w3c/vc-data-model)
+TBD
 
 ## Install
 
-Requires [Node.js][] 8.3+
+Requires [Node.js][] 18+
 
 To install via [NPM][]:
 
 ```
-npm install credentials-context
+npm install @digitalbazaar/credentials-context
 ```
 
 ## Usage
 
 ```js
-import credentialsContext from 'credentials-context';
+import {contexts, contextsMetadata} from 'credentials-context';
 // or
-const credentialsContext = require('credentials-context');
-
-// use URL in a JSON-LD context
-const obj = {
-  "@context": [
-    credentialsContext.CONTEXT_URL_V1,
-    // ...
-  ],
-  // ...
-};
-
-// get context data for a specific context
-const data = credentialsContext.CONTEXT;
-// ...
+const {contexts, contextsMetadata} = require('credentials-context');
 ```
 
+The `contexts` [Map][] can be used to access individual contexts by id or load
+them into a JSON-LD document loader. The `contextsMetadata` [Map][] contains
+extra information about each context.
+
 This package can be used with bundlers, such as [webpack][], in browser
 applications.
 
 ## API
 
 The library exports two properties:
-- `constants`: A Object that maps constants to well-known context URLs. The
-  main constant `CONTEXT_URL_V1` may be updated from time to time to the
-  latest context location.
-- `contexts`: A `Map` that maps URLs to full context data.
+- `contexts`: A [Map][] associating context URLs to context data.
+- `contextsMetadata`: A [Map][] associating context URLs to context metadata.
 
-## Contribute
+Note that the `contextsMetadata` format is experimental and subject to change.
 
-Note: If editing the Readme, please conform to the
-[standard-readme](https://github.com/RichardLitt/standard-readme) specification.
+The context files are available in the published `contexts/` directory. The
+metadata has a URL for each context. Note that these files are semantically
+equivalent to published spec contexts but the formatting may differ such that
+strict file digests are not equivalent.
+
+## Status
+
+The following contexts are available as of mid-2024. They track the published
+spec contexts.
+
+- `https://www.w3.org/2018/credentials/v1`: Stable.
+- `https://www.w3.org/ns/credentials/v2`: Under development.
+- `https://www.w3.org/ns/credentials/undefined-terms/v2`: Under development.
+
+## Developing
+
+Source is available at:
+- https://github.com/digitalbazaar/credentials-context
+
+**WARNING**: The `.jsonld` files in `contexts/` is auto-generated by the `npm
+run build` script each time the test suite is run.
+
+DO NOT edit them directly since changes will be overwritten.
+
+Make all context changes to `lib/*.js`.
```

Comment:
Still not a fan of the JSON-LD embedded in JS turned back into JSON-LD approach...but now is not the time to change it (alas...).

Reply:
Yes, it's terrible.  However, I think this design allows that to easily change.  The best answer is using import attributes.  That spec is not stage 4 yet, so there are issues with using it.  The original context packages used browser+webpack and node specific code to load json files, but that got removed for reasons.  It might return. 
 We will revisit this later.